pkg/ccl/changefeedccl/kvfeed/kv_feed.go

```diff
@@ -215,8 +245,21 @@ func (f *kvFeed) run(ctx context.Context) (err error) {
 			return err
 		}
 
+		if f.initialScanOnly {
```

Comment:
Would it work to just treat `initialScanOnly` as `EndTime == initialHighWater` (when we have `withInitialBackfill`) and not need the extra parameter at all?

Reply:
When we indicate `initialScanOnly`, we do not want the rangefeed to start after the initial scan has run. However, if we were to go through with the idea you proposed, the rangefeed would turn on as we would reach `runUntilTableEvent`. This is why I think we need the two parameters.

Reply:
I don't necessarily mean removing the if statement against running `runUntilTableEvent`, just mean not needing the extra `InitialScanOnly` property that gets passed around and instead checking if your highwater is already at the endtime

Reply:
ahh ok, yeah I can go through with that instead, thanks for the suggestion!